apps/passport-server/src/services/telegramService.ts

```diff
@@ -199,12 +293,10 @@ export class TelegramService {
             }
           );
 
-          menu.webApp("Generate ZKP 🚀", proofUrl);
```

Comment:
@rrrliu turns out setting the menu here was causing the duplication. So now it needs to be set using the `dynamic()` function in the constructor and `this.proofUrl`